controllers/memberController.js

```diff
@@ -84,7 +84,26 @@ const requestConfirmEmail = (req, res) => {
     }
 }
 
+// Delete account. Called by script in member page
+const requestDeleteAccount = (req, res) => {
+    const usernameLowercase = req.params.member.toLowerCase();
+
+    // Load their case sensitive username
+    const username = getUsernameCaseSensitive(usernameLowercase);
+    if (!username) return res.status(404).json({message: 'Member not found'})
+
+    // Check to make sure that they are the same person requesting deletion
+    if (req.user === usernameLowercase) {
+        removeMember(username);
```

Comment:
This needs to be usernameLowercase, not username. Otherwise `removeMember()` won't be able to find them in the json data.